tvb_scripts/timeseries/model.py

```diff
@@ -110,7 +110,6 @@ def __init__(self, data, **kwargs):
                     "and labels_dimensions!: %s" % (self._tvb.labels_ordering, self._tvb.labels_dimensions.keys()))
 
         self._tvb.configure()
-        self.configure_time()
```

Comment:
Why is this line removed?

Reply:
This line has been removed only for passing over some code, without throwing errors. The errors have been solved and the correct code can be seen in a future commit.